python/paddle/fluid/initializer.py

```diff
@@ -729,31 +729,32 @@ class BilinearInitializer(Initializer):
 
         .. code-block:: python
 
-            import paddle.fluid as fluid
+            import paddle
```

Comment:
Suggest to change import order. Usually we group the imports by function, then sort it by alphabetical order. Here `import math` is in the middle of import paddle, which is ugly.

Suggest to change like this:

```
import math

import paddle
import paddle.nn
from paddle.regularizer import L2Decay
```

Reply:
I've already modified.

Reply:
Could you just put in order? You change it like:

import xxx
from xxx import
import xxx

Why can't you just make it pretty? Since all xxx are `paddle`

import xxx
import xxx
from xxx import